projects/wave-core/src/lib/operators/dialogs/data-repository/raster/source-dataset.component.ts

```diff
@@ -34,7 +34,7 @@ export class SourceDatasetComponent implements OnInit {
     _showPreview = false;
     _showDescription = false;
     _channelSource: ChannelDataSource;
-    _displayedColumns = ['name', 'measurement'];
+    _displayedColumns = ['name', 'measurement', 'start', 'end'];
```

Comment:
You should display `start` and `end` only if there is information available.

Reply:
fixed in bc0236c100809397960ca46b0f1e76b973d72473